CoronaTracker.Core/Models/Countries/Country.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace CoronaTracker.Core.Models.Countries
+{
+    public class Country
+    {
+        [Key]
+        public string CountryName { get; set; }
```

Comment:
We still want to have an ID. that is `Guid` but when we upsert we will search by the country name or ISO name.